tests/integration/features.bats

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	# It does not make sense to repeat these trivial tests for non-root.
+	# Also, they fail due to $ROOT not being set and XDG_RUNTIME_DIR
+	# pointing to another user's directory after sudo rootless.
+	requires root
+}
+
+@test "features [idmap]" {
+	runc features
+	[ "$status" -eq 0 ]
+	enabled=$(echo "$output" | jq '.linux.mountExtensions.idmap.enabled')
+	[[ "$enabled" == "true" ]]
+}
```

Comment:
This doesn't seem to worth testing

Reply:
Oh, I was thinking to move this to unit tests. But sure, I'll remove it! :)